pytorch_lightning/trainer/trainer.py

```diff
@@ -483,12 +484,24 @@ def __init__(
                     profiler = AdvancedProfiler()
                     trainer = Trainer(profiler=profiler)
 
+            benchmark (bool): If true enables cudnn.benchmark. From PyTorchdocs: "This flag allows you to enable the inbuilt cudnn auto-tuner 
```

Comment:
you may rather use citations, see https://docutils.sourceforge.io/docs/user/rst/quickref.html#citations

Reply:
will brought this line in..
I'm not on the side of my computer at this moment, so I can't bring a fix.